src/ensembl/src/shared/image-button/ImageButton.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ImageButton, { ImageButtonStatus } from './ImageButton';
+
+describe('<ImageButton />', () => {
+  it('renders without error', () => {
+    expect(() => {
+      mount(<ImageButton />);
+    }).not.toThrow();
+  });
+
+  describe('prop buttonStatus', () => {
+    it('has a buttonStatus set by default', () => {
+      const wrapper = mount(<ImageButton />);
+
+      expect(wrapper.props().buttonStatus).toEqual(ImageButtonStatus.DEFAULT);
+    });
+
+    it('respects the buttonStatus prop', () => {
+      const wrapper = mount(
+        <ImageButton buttonStatus={ImageButtonStatus.ACTIVE} />
+      );
+
+      expect(wrapper.props().buttonStatus).toEqual(ImageButtonStatus.ACTIVE);
+    });
+  });
+
+  describe('prop description', () => {
+    it('has a description set by default', () => {
+      const wrapper = mount(<ImageButton />);
+
+      expect(wrapper.props().description).toEqual('');
+    });
+
+    it('respects the description prop', () => {
+      const wrapper = mount(<ImageButton description={'foo'} />);
+
+      expect(wrapper.props().description).toEqual('foo');
+    });
+  });
+
+  describe('prop image', () => {
+    it('has a image set by default', () => {
+      const wrapper = mount(<ImageButton />);
+
+      expect(wrapper.props().image).toEqual('');
+    });
+
+    it('respects the image prop', () => {
+      const wrapper = mount(<ImageButton image={'foo.svg'} />);
+
+      expect(wrapper.props().image).toEqual('foo.svg');
+    });
+  });
```

Comment:
For the three `describe` blocks above, what is the intended behaviour for one or the other prop? It's not terribly interesting to know that there is a particular default prop that gets replaced with a prop that is passed to a component — it is more interesting to know why we are passing the props, i.e. how these props are used by the component.

Reply:
The tests are now updated.